reactioncommerce/reaction
# Fix Email Notification During Order Refund

**Resolves**
#1881 

**Test**
1. Place an order
1. Approve and capture the order
1. Try to issue a refund for more than order total
1. Observe that email notification sent toast message is not shown.
